### PR TITLE
add kn quickstart 0.25

### PR DESCRIPTION
**Pull request by csantanapr**
add `kn quickstart` to getting started guide for version 0.25

### CODE REVIEW

**Review by psschwei — APPROVED** [review]
/lgtm
@csantanapr I didn't see the "site preview" option, but I'm assuming that's because you had to do the cherry-pick manually. Putting a `/hold` in the small chance that I'm fulfilling the cliche by assuming, but feel free to remove as you see fit.
/hold

File: docs/snippets/install-kn.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
 !!! todo "Installing the `kn` CLI"
 
     === "Using Homebrew"
-        For macOS, you can install `kn` by using <a href="https://github.com/knative/homebrew-client" target="_blank">Homebrew</a>.
+        For macOS, you can install `kn` by using [Homebrew](https://brew.sh"){target=_blank}.
```
Comment (psschwei) [inline docs/snippets/install-kn.md:4]: Need to delete the quote after `brew.sh`
/lgtm cancel

**Review by psschwei — CHANGES_REQUESTED** [review]

File: docs/snippets/install-kn.md (hunk 1 above)
Comment (csantanapr) [reply docs/snippets/install-kn.md:4]: 😄 I just copy the file from the one you created https://github.com/knative/docs/blob/mkdocs/docs/snippets/install-kn.md
I guess we need to do another PR to fix in mkdocs branch

**Comment by csantanapr** [comment]
yeah site preview it currently doesn't work on other branches

**Comment by csantanapr** [comment]
I just enable deploys for branch `release-0.25` but not sure when it will take affect if it needs a new PR

**Review by psschwei — APPROVED** [review]
/lgtm
Holding for preview (feel free to cancel whenever)
/hold

**Comment by csantanapr** [comment]
/unhold
yeah it looks like preview is not working for none production branches

**Pull request merged**